packages/rich-text/src/component/use-space.js

```diff
@@ -21,13 +21,38 @@ export function useSpace() {
 				return;
 			}
 
-			const { keyCode, altKey, metaKey, ctrlKey } = event;
+			const { keyCode, altKey, metaKey, ctrlKey, key } = event;
 
 			// Only consider the space key without modifiers pressed.
 			if ( keyCode !== SPACE || altKey || metaKey || ctrlKey ) {
```

Comment:
Should we remove `keyCode !== SPACE`, or will that cause problems?

Reply:
Right now I just patched the existing approach, so I wouldn't touch that condition. But ideally we would ditch `keyCode` altogether and rely mostly on `key`, which is the [main non-deprecated property](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) for this stuff. Then we'll know what shortcomings `key` presents, and we could mitigate them with properties like `code`, `location`, etc.

Reply:
We should also mind the [composition events](https://developer.mozilla.org/en-US/docs/Web/API/Element/compositionstart_event).

Reply:
I was just reading this interesting thread on `keydown` and `event.key` - https://twitter.com/jaffathecake/status/1480493537112248325.